packages/react-log-viewer/src/LogViewer/LogViewer.tsx

```diff
@@ -29,6 +29,8 @@ interface LogViewerProps {
   scrollToRow?: number;
   /** Number of rows to display in the log viewer */
   itemCount?: number;
+  /** Flag indicating that log viewer is wrapping text or not */
+  wrapText?: boolean;
```

Comment:
Typically in PF we start boolean prop names with "is" or "has'.

Reply:
If you really want to prefix it you probably also need to change the wording a bit to accommodate
- isTextWrapped
- hasWrappedText
- shouldWrapText